packages/material-ui/src/ButtonBase/TouchRipple.test.js

```diff
@@ -1,87 +1,119 @@
 import React from 'react';
 import { useFakeTimers } from 'sinon';
-import { assert } from 'chai';
-import { createShallow, createMount, getClasses } from '@material-ui/core/test-utils';
+import { expect } from 'chai';
+import { createMount, getClasses } from '@material-ui/core/test-utils';
+import { cleanup, createClientRender } from 'test/utils/createClientRender';
 import describeConformance from '../test-utils/describeConformance';
-import Ripple from './Ripple';
 import TouchRipple, { DELAY_RIPPLE } from './TouchRipple';
 
 const cb = () => {};
 
 describe('<TouchRipple />', () => {
-  let shallow;
-  let mount;
   let classes;
+  let mount;
+  const render = createClientRender({ strict: true });
+
+  /**
+   * @param {object} other props to spread to TouchRipple
+   */
+  function renderTouchRipple(other) {
+    const touchRippleRef = React.createRef();
+    const { container } = render(
+      <TouchRipple
+        ref={touchRippleRef}
+        classes={{
+          ripple: 'ripple',
+          rippleVisible: 'ripple-visible',
+          child: 'child',
+          childLeaving: 'child-leaving',
+        }}
+        {...other}
+      />,
+    );
+
+    return {
+      instance: touchRippleRef.current,
+      queryAllActiveRipples() {
+        return container.querySelectorAll('.ripple-visible .child:not(.child-leaving)');
```

Comment:
These selectors were always coupled with the render input so they should live next to it. It follows the same approach the the base `render` from `@testing-library/react` has by binding the selectors (`getByRole`, `getByLabelText` etc.) to the base element.